src/eos-updater-fetch.h

```diff
@@ -23,8 +23,8 @@
 #pragma once
 
 #include "eos-updater-generated.h"
-#include "eos-updater-util.h"
-#include <ostree.h>
+
+#include <gio/gio.h>
```

Comment:
Yeah, we rely on that. Actually `eos-updater-generated.h` is where `EosUpdater` is declared.